src/pages/management/configuration-api/index.mdx

```diff
@@ -2810,9 +2810,9 @@ Here's what you need to know about **webhooks**:
 
 ### Register Webhook
 
-Registers a webhook for the Client ID (application) provided in the request. One Client ID can register up to three webhooks for every `action`. To receive `license` webhooks, you need to [enable them](#enable-license-webhooks) after registration.
+Registers a webhook for the Client ID (application) provided in the request. One Client ID can register up to three webhooks for every `action`. The `license` webhooks'll be enabled automatically. The [Disable License Webhooks](#disable-license-webhooks) method should be called explicitly when there is a need to disable some webhooks.
```

Comment:
I'm not sure if we should mention Disable License Webhooks here - it seems a little extra. But it's your call :)